frontend/src/Settings/ImportLists/ImportLists/EditImportListModalContent.js

```diff
@@ -58,7 +59,7 @@ function EditImportListModalContent(props) {
   return (
     <ModalContent onModalClose={onModalClose}>
       <ModalHeader>
-        {id ? translate('EditImportList') : translate('AddImportList')}
+        {`${id ? translate('EditImportList') : translate('AddImportList')} - ${implementationName}`}
```

Comment:
For metadata we use `Edit {metadataType} Metadata`, should that be changed to a similar format?

I think regardless the implementation name should be part of the translation so the full context of how it's used is presented in the translation in the event that affects the translation in other language.

Reply:
It gives additional information on how the translation is being used, including the secondary title being appended, which could affect how it's supposed to be translated. In a RTL language it's also necessary because the display would be wrong, whether that's more wrong than mixing a proper English name in RTL, I'm not sure.

Reply:
Yeah, having the implementation name at the end like that still, just wanted to make sure it was part of the translation.